src/bin/build-bot.rs

```diff
@@ -0,0 +1,43 @@
+extern crate irc;
+
+use irc::client::prelude::*;
+use std::default::Default;
+use std::env;
+
+fn main() {
+    let repository_slug = env::var("TRAVIS_REPO_SLUG").unwrap();
+    let branch = env::var("TRAVIS_BRANCH").unwrap();
+    let commit = env::var("TRAVIS_COMMIT").unwrap();
+    let commit_message = env::var("TRAVIS_COMMIT_MESSAGE").unwrap();
+
+    let config = Config {
+        nickname: Some("irc-crate-ci".to_owned()),
+        server: Some("irc.pdgn.co".to_owned()),
+        use_ssl: Some(true),
+        ..Default::default()
+    };
+
+    let mut reactor = IrcReactor::new().unwrap();
+    let client = reactor.prepare_client_and_connect(&config).unwrap();
+    client.identify().unwrap();
+
+    reactor.register_client_with_handler(client, move |client, message| {
+        match message.command {
+            Command::Response(Response::RPL_ISUPPORT, _, _) => {
+                client.send_privmsg(
+                    "#commits",
+                    format!(
+                        "Hello from Travis CI! {}/{} {}: {}",
```

Comment:
It would be nice if we could change the message to be a bit more readable by cutting the commit hash to 7 or 8 characters.

Something along the lines of a message like:

>[Bassetts/irc/ci-bot] (f1d7d714) Add test bot for CI

I don't think we really need the hello from Travis CI part since we have the other information there. 😄

Reply:
@aatxe thanks for the feedback. I was thinking the full commit hash was a bit much, I will get right on that.